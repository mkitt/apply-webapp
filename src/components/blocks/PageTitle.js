// @flow
import React, { type Node } from 'react'
import glamorous from 'glamorous'
import SectionContainmentBase from '../elements/SectionContainment'
import Title from '../elements/Title'

type Props = {
  children: Node,
}

export default ({ children, ...props }: Props) => (
  <SectionContainment {...props}>
    <Title>
      {children}
    </Title>
  </SectionContainment>
)

const SectionContainment = glamorous(SectionContainmentBase)({
  marginBottom: 6,
})
