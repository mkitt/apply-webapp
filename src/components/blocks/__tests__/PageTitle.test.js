import React from 'react'
import renderer from 'react-test-renderer'
import PageTitle from '../PageTitle'

describe('<PageTitle />', () => {
  it('shows the correct snapshot tree for the <PageTitle> block', () => {
    const component = <PageTitle>The Page Title</PageTitle>
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
