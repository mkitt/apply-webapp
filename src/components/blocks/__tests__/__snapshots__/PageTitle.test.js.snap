// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageTitle /> shows the correct snapshot tree for the <PageTitle> block 1`] = `
.css-0,
[data-css-0] {
  position: relative;
  max-width: 960px;
  margin-right: auto;
  margin-left: auto;
  padding-right: 16px;
  padding-left: 16px;
  margin-bottom: 128px;
}

.css-1,
[data-css-1] {
  font-size: 48px;
  font-weight: 200;
  color: #808080;
}

<section
  className="css-0"
>
  <h1
    className="css-1"
  >
    The Page Title
  </h1>
</section>
`;
